browser/model/virtualbox.js

```diff
@@ -131,6 +131,9 @@ class VirtualBoxInstallWindows extends VirtualBoxInstall {
           return this.configure(installer);
         })
         .then((result) => {
+          if(result) {
```

Comment:
This check is not required, installation ether succeeded and then callback is executed or failed and catch callback is executed.